package.json

```diff
@@ -37,8 +37,7 @@
     "eslint-plugin-react-hooks": "4.0.7",
     "kind-of": "6.0.3",
     "node-fetch": "2.6.1",
-    "node-notifier": "^9.0.0",
-    "yargs-parser": "^18.1.1"
```

Comment:
Adds a component governance issue, but unbreaks wdio cli. Need to talk to Jon about a solution.